src/components/Autocomplete/README.md

```diff
@@ -301,6 +301,123 @@ class AutocompleteExample extends React.Component {
 }
 ```
 
+### Autocomplete with lazy loading
+
+```jsx
+class AutoCompleteLazyLoadExample extends React.Component {
+  paginationInterval = 25;
+
+  options = Array.from(Array(1000)).map((_, index) => ({
+    value: `rustic ${index}`,
+    label: `Rustic ${index}`,
+  }));
+
+  state = {
+    selected: [],
+    inputText: '',
+    options: this.options,
+    visibleOptionIndex: this.paginationInterval,
+  };
+
+  render() {
+    const {visibleOptionIndex, selected, options} = this.state;
+    const textField = (
+      <Autocomplete.TextField
+        onChange={this.updateText}
+        label=""
```

Comment:
It's probably good to include a label in our examples, or at least a hidden one with an accessibility label provided. That way we're illustrating a11y best practices.